newIDE/app/src/SceneEditor/Toolbar.js

```diff
@@ -65,8 +64,7 @@ const Toolbar = (props: Props) => {
         canDeleteSelection={
           props.instancesSelection.getSelectedInstances().length !== 0
         }
-        canRenameObject={props.canRenameObject}
```

Comment:
is this not relevant anymore to be able to disable the action?

Reply:
I could add something but the worst case is that we try to rename an object that is the first element of a list, that would be `undefined` if the list is empty. Both methods handle that case. Do you think I should add this?